module-2/backend/week_one.md

```diff
@@ -5,28 +5,52 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  `GET = reads data, Post = creates, PUT = update/replace, PATCH = update/modify, DELETE = delete`
+  
 2. What is Sinatra?
+  `Sinatra is a web development framework`
 4. What is MVC?
+  `model, view, controller.  A file artchitecture that seperates responsabilities the model deals with the database, the controller interacts with the client and the database, and the view is what is delivered to the client`
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+`because Sinatra requires us to follow these conventions in order to create relationships and direct the flow of data`

```

Comment:
Sinatra actually doesn't require us to - we could create whichever combination we'd like. we could also name our paths whatever we'd like. we follow these conventions (called RESTful conventions) because these are the agreed upon standards/conventions by most developers. Rails also makes some assumptions based on these conventions. We have to manually follow these conventions in Sinatra because Sinatra isn't picky at all about our route naming choices